features/login-testcase.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Login testcase. 
+
+    Positive scenario (passing valid login and password).
+
+    Scenario: User opens pah-fm website.
+       Given User has account in the system already created by system admin.
+        When User opens pah-fm website
+        Then User sees login view with 2 text fields: "Username" and "Password" and 1 button "Login".
```

Comment:
This is also bad habit - using mixed whens and thens. Please try to split the scenarios for small ones, where we have structure:

```
Given
When
Then
```

not

```
Given
When
Then
When
Then
```

Reply:
I am aware that this might not be clear at this point, although for the case of the automated tests - try to make them short and test one thing.